db/migrate/20180411045246_change_installation_date_in_stations.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeInstallationDateInStations < ActiveRecord::Migration[5.1]
+  def change
+    change_column :stations, :installation_date, :string
```

Comment:
Sorry @hmesander ! But the date object couldn't handle 8/13/2013 to 8/15/2013 for some unknown reason... Once I changed this field to string everything worked. I am happy to change it back if we can figure out why those dates broke the seed.

Reply:
This looks great--can't wait to deploy!